netboot/main.go

```diff
@@ -108,6 +110,16 @@ func main() {
 		}
 		for _, d := range dhcp {
 			if err := boot(iface.Name, d); err != nil {
+				if !*noFixmynetboot {
+					cmd := exec.Command("fixmynetboot")
```

Comment:
By default I think we loop over all available interfaces. However, `fixmynetboot` only checks `eth0`. So this may call `fixmynetboot` several times for `eth0`.

Reply:
Good catch! I've updated the code to run only for the failing interface